nixos/modules/services/x11/window-managers/exwm.nix

```
@@ -19,6 +19,18 @@ in
   options = {
     services.xserver.windowManager.exwm = {
       enable = mkEnableOption "exwm";
+      loadScript = mkOption {
```

I think this is slightly more accurate, and will probably avoid some confusion.

Not to mention it's more aligned with other "init" options (e.g. `programs.zsh.loginShellInit`)
```suggestion
      loadScriptInit = mkOption {
```

---

"init" has a special meaning in Emacs though, and I think that might confuse things.